java/benchmarks/src/main/java/glide/benchmarks/BenchmarkingApp.java

```diff
@@ -66,36 +66,66 @@ private static Options getOptions() {
         // create the Options
         Options options = new Options();
 
-        options.addOption(Option.builder("help").desc("print this message").build());
+        options.addOption(Option.builder("h").longOpt("help").desc("Print this message").build());
```

Comment:
Are we going to update this message later? Or is it meant to just be a dummy message? If we want to update this later, let's make sure that it doesn't fall through the cracks.

Reply:
A dummy message